include/deal.II/grid/grid_reordering.h

```diff
@@ -668,7 +668,8 @@ class GridReordering
   static void
   invert_all_cells_of_negative_grid(
     const std::vector<Point<spacedim>> &all_vertices,
-    std::vector<CellData<dim>> &        original_cells);
+    std::vector<CellData<dim>> &        original_cells,
+    const bool                          use_new_style_ordering = false);
```

Comment:
If you want to introduce the new variable, you should document it. In two releases, nobody will now the difference between old and new style. Should we switch the default value to true so that only people who need backward compatibility have to deal with negative volume?

Reply:
I'll add the documentation. However, the problem with changing the default argument here is that this function already expects the cells to use the old ordering, so it really does need to be `false`.

I'm hoping to get rid of all this compatibility stuff before the release and move this function into `GridTools` - see also #8562.